articles/active-directory-b2c/partner-nok-nok.md

```diff
@@ -12,54 +12,56 @@ ms.date: 06/21/2024
 
-* An Azure subscription
-  * If you don't have one, get a [Azure free account](https://azure.microsoft.com/free/)
-* An Azure AD B2C tenant linked to the Azure subscription
-  * [Tutorial: Create an Azure Active Directory B2C tenant](tutorial-create-tenant.md) 
-* Go to [noknok.com](https://noknok.com/).
-  * On the top menu, select **Demo**.
+* An Azure subscription. If you don't have one, get an [Azure free account](https://azure.microsoft.com/free/)
+* An Azure AD B2C tenant linked to the Azure subscription [Tutorial: Create an Azure Active Directory B2C tenant](tutorial-create-tenant.md)
+* Nok Nok Cloud evaluation tenant to be used for registration and authentication
 
 ## Scenario description
 
-To enable passwordless FIDO authentication for your users, enable Nok Nok as an identity provider (IdP) in your Azure AD B2C tenant. The Nok Nok solution integration includes the following components:
+To enable Passkey authentication for your users, enable Nok Nok as an identity provider (IdP) in your Azure AD B2C (AAD B2C) tenant. The Nok Nok integration includes the following components:
 
 * **Azure AD B2C** – authorization server that verifies user credentials
-* **Web and mobile applications** – mobile or web apps to protect with Nok Nok solutions and Azure AD B2C
-* **Nok Nok app SDK or Passport app** – authenticate Azure AD B2C enabled applications. 
-  * Go to the Apple App Store for [Nok Nok Passport](https://apps.apple.com/us/app/nok-nok-passport/id1050437340)
-  * Or, Google Play [Nok Nok Passport](https://play.google.com/store/apps/details?id=com.noknok.android.passport2&hl=en&gl=US)
+* **Web and mobile applications** – mobile or web apps to protect with Nok Nok solutions and AAD B2C
+* **Nok Nok Tutorial Web App** – application used to register the passkey on your device
+* **Nok Nok Sign in App** – application used for authenticating AAD B2C applications using passkey.
 
-The following diagram illustrates the Nok Nok solution as an IdP for Azure AD B2C by using OpenID Connect (OIDC) for passwordless authentication.
 
-   ![Diagram of Nok Nok as IdP for Azure AD B2C using OpenID Connect (OIDC) for passwordless authentication.](./media/partner-nok-nok/nok-nok-architecture-diagram.png)
+The following diagram illustrates the Nok Nok solution as an IdP for AAD B2C by using OpenID Connect (OIDC) for Passkey authentication.
+
+![Diagram of Nok Nok as IdP for AAD B2C using OpenID Connect (OIDC) for Passkey authentication.](./media/partner-nok-nok/nok-nok-architecture-diagram.png)
+
+### Scenario 1: Passkey Registration
+1. The user navigates to the Nok Nok Tutorial Web App using the link provided by Nok Nok.
+2. They enter their AAD B2C username and default tutorial app password.
+3. They are prompted to register the passkey.
+4. The Nok Nok server validates the passkey credential and confirms successful registration to the user.
+5. The passkey is now registered on the user's device and is ready to authenticate.

```

Comment:
Suggestion to change passive voice to active voice:
Nok Nok registration on the user's device is ready to authenticate.